@here/olp-sdk-dataservice-api/lib/lookup-api.ts

```diff
@@ -165,7 +167,7 @@ export async function resourceAPI(
 export async function resourceAPIList(
     builder: RequestBuilder,
     params: { hrn: string; region?: string }
-): Promise<API[] | ApiNotFoundError> {
+): Promise<Response> {
```

Comment:
Please use deprecation instead of changes to avoid API breaks

Reply:
Done